testing/todocheck_test.go

```diff
@@ -491,6 +491,21 @@ func TestConfigAutoDetectWithSSHGitConfig(t *testing.T) {
 				WithType(errors.TODOErrTypeMalformed).
 				WithLocation("scenarios/auto_detect_config/main.go", 3).
 				ExpectLine("// TODO - malformed todo")).
+		ExpectTodoErr(
+			scenariobuilder.NewTodoErr().
+				WithType(errors.TODOErrTypeIssueClosed).
+				WithLocation("scenarios/auto_detect_config/main.go", 5).
+				ExpectLine("// TODO #1: closed issue")).
+		ExpectTodoErr(
+			scenariobuilder.NewTodoErr().
+				WithType(errors.TODOErrTypeIssueClosed).
+				WithLocation("scenarios/auto_detect_config/main.go", 7).
+				ExpectLine("// TODO 2: closed issue")).
+		ExpectTodoErr(
+			scenariobuilder.NewTodoErr().
+				WithType(errors.TODOErrTypeNonExistentIssue).
+				WithLocation("scenarios/auto_detect_config/main.go", 9).
+				ExpectLine("// TODO #9999999: non-existent issue")).
```

Comment:
Instead of modifying the existing `TestConfigAutoDetectWithSSHGitConfig` test, please create a new one where you're testing your new feature in isolation.

Currently, if this test fails, we won't know if it's because of an issue with the issue tracker auto-detection feature or the github issues with hashtag feature.